common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2083,6 +2084,14 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(16)
 
+  val WORKER_STORAGE_EXPIRE_DIR_TIMEOUT: ConfigEntry[Long] =
+    buildConf("celeborn.worker.storage.expireDirs.timeout")
+      .categories("worker")
+      .version("0.3.1")
+      .doc(s"The timeout for a expire dirs to be deleted on disk.")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("6h")
```

Comment:
Why is the default value of `celeborn.worker.storage.expireDirs.timeout` 6h?

Reply:
When the execution time of most tasks exceeds the expiration time, the possibility of accidental deletion increases. In our production environment, we set it to 12 hours.

Reply:
@wilsonjie, could this config have no default value?

Reply:
Can it be consistent with the expiration time of hdfs dir ?   celeborn.master.hdfs.expireDirs.timeout=1h

Reply:
@wilsonjie, yes.